node-hub/dynamixel-client/main.py

```diff
@@ -0,0 +1,198 @@
+"""
+Dynamixel Client: This node is used to represent a chain of dynamixel motors. It can be used to read positions,
+velocities, currents, and set goal positions and currents.
+"""
+
+import os
+import time
+import argparse
+import json
+
+import pyarrow as pa
+
+from dora import Node
+
+from common.dynamixel_bus import DynamixelBus, TorqueMode
+
+
+class Client:
+
+    def __init__(self, config: dict[str, any]):
+        self.config = config
+
+        description = {}
+        for i in range(len(config["ids"])):
+            description[config["joints"][i]] = (config["ids"][i], config["models"][i])
+
+        self.bus = DynamixelBus(config["port"], description)
+
+        # Set client configuration values
+
+        try:
+            self.bus.write_torque_enable(config["torque"], self.config["joints"])
+        except Exception as e:
+            print("Error writing torque status:", e)
```

Comment:
Should not being able to write torque raise an error?

It seems that this would make debugging a non-moving arm hard.